scripts/pre_commit_linter.py

```diff
@@ -268,6 +268,14 @@
 
 CODEOWNER_FILE_PATHS = ['./app.yaml', './manifest.json']
 
+CODEOWNER_IMPORTANT_PATHS = [
```

Comment:
Can you please add a comment here and also in the CODEOWNERS file saying that this need to be in sync?

Reply:
Yeah sure. Will do that!